gym/utils/closer.py

```diff
@@ -1,68 +0,0 @@
-import atexit
-import threading
-import weakref
-
-
-class Closer:
```

Comment:
I see you have removed `Closer`, which I recall is used in some cases such as in #2316. @XuehaiPan would you mind reviewing this?

Reply:
That's fine. The legacy `Closer` can be implement with `__del__`:

```
class MyClass:
    def close(self):
        ...
    
    def __del__(self):
        self.close()
```